server/src/main/java/org/opensearch/action/search/MultiSearchRequest.java

```diff
@@ -272,6 +273,9 @@ public static void readMultiLineFormat(BytesReference data,
                             allowNoIndices = value;
                         } else if ("ignore_throttled".equals(entry.getKey()) || "ignoreThrottled".equals(entry.getKey())) {
                             ignoreThrottled = value;
+                        } else if ("cancel_after_timeinterval".equals(entry.getKey()) ||
+                            "cancelAfterTimeinterval".equals(entry.getKey())) {
```

Comment:
cancel_after_timeinterval -> cancel_after_time_interval

The term `timeinterval` should probably be in separate words, as you have done intuitively in all the camelCase ocurrences except here:) 

```suggestion
                        } else if ("cancel_after_time_interval".equals(entry.getKey()) ||
                            "cancelAfterTimeInterval".equals(entry.getKey())) {
```

Reply:
For `SearchRequest` parameters I was treating `timeinterval` as single word to avoid adding another separator and camel casing may not make sense for it. However,  in MSearch I see it supports parameters both with and without separators, so to be consistent I kept `Timeinterval` again as single word. Do you think the parameter is confusing in current form `cancel_after_timeinterval` ? I am also introducing a cluster setting for setting a global cancellation timeout (in part 2) and was planning to again name it as `search.cancel_after_timeinterval`

Reply:
I am not sure what the right name is, but it must be consistent between the underscore and the camelCase version, so either `cancel_after_timeinterval` and `cancelAfterTimeinterval` or `cancel_after_time_interval` and `cancelAfterTimeInterval`.

Reply:
I agree about the consistency but before making the change was looking for some inputs on deciding on the name itself.